specs/waku/waku-v2.md

```diff
@@ -209,7 +221,7 @@ corresponds to topics in Waku v1.
 
 A node that only sets this field but doesn't subscribe to any topic SHOULD only
 get notified when the content subtopic matches. A content subtopic matches when
-a message `contentTopic` is the same. This means such a node acts as a light node.
+a `Wakumessage` `contentTopic` field is the same. This means such a node acts as a light node.
```

Comment:
```suggestion
a `Wakumessage` `contentTopic` field is the same.
```

`This means such a node acts as a light node.` is not correct.
It can be a full node that is only interested in notifications on a specific `contentTopic` for its application, and just relays all the rest.